cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/runtimejob/DataprocRuntimeJobManager.java

```diff
@@ -201,6 +202,10 @@ public void launch(RuntimeJobInfo runtimeJobInfo) throws Exception {
       LOG.debug("Successfully submitted hadoop job {} to cluster {}.", job.getReference().getJobId(), clusterName);
       DataprocUtils.emitMetric(provisionerContext, region,
                                "provisioner.submitJob.response.count");
+    } catch (AlreadyExistsException ex) {
+      //the job id already exists, ignore the job.
+      LOG.warn("The job already exists: {}", ex.getMessage());
```

Comment:
should we make it debug?

Reply:
`The dataproc job {} already exists. Ignoring resubmission of the job`

Reply:
The reason I want to put it at warn level is to catch attention, because it could be:
1. The app fabric got restarted somehow and we issued a re-submission of the job.
2. There are some bug in our code that caused a re-submission of the job.
what do you think ?

I've revised the message accordingly:)

Reply:
I see. Though in both cases, the system will recover itself and user does not need to take any action. Thats why I was suggesting to use debug level.